anthos-vmruntime/README.md

```diff
@@ -0,0 +1,405 @@
+based on which the Anthos on bare metal cluster was setup, automatically creates
+an [`Ingress` resource named `pos-ingress`](https://github.com/GoogleCloudPlatform/anthos-samples/blob/kubevirt-guide/anthos-bm-gcp-terraform/resources/manifests/pos-ingress.yaml).
+This resource routes the traffic from the public IP address of the Ingress
+loadbalancer to the _api server service_ of the Point Of Sale sample application.
+
+```sh
+kubectl describe ingress/pos-ingress
+```
+```sh
+# expected output
+Name:             pos-ingress
+Labels:           <none>
+Namespace:        default
+Address:          34.117.58.199
+Default backend:  default-http-backend:80 (<error: endpoints "default-http-backend" not found>)
+Rules:
+  Host        Path  Backends
+  ----        ----  --------
+  *
+              /   api-server-svc:8080 (<error: endpoints "api-server-svc" not found>)
+Annotations:  <none>
+Events:       <none>
+```
+
+Note that there are erros indicating that the `api-server-svc` was not found.
+This is because we deleted the resources created by default in [an earlier step](#cleanup-the-resources-created-by-default-in-the-installation-guide).
+We will have to re-create this `Service` pointing to the
+`VirtualMachineInstance`. This way, we can get the `Ingress` working once again
+and reach the sample application inside the VM via the `Ingress` LB IP.
+
+Copy the [`pos-service.yaml`](pos-service.yaml) file into the admin workstation
+VM. Then, update the file to include the IP address of the VM as a service
+`Endpoint`. Finally apply the changes to the cluster.
+
+```sh
+# you should have the pos-service.yaml copied into the admin workstation
+
+# retrieve the IP address of the virtual machine we created
+export VM_IP=$(kubectl get vmi/pos-vm -o jsonpath='{.status.interfaces[0].ipAddress}')
+
+# update the pos-service.yaml with the virtual machine's IP address
+sed -i "s/VM_IP/${VM_IP}/g" pos-service.yaml
+
+# apply the changes to the Anthos on bare metal cluster
+kubectl apply -f pos-service.yaml
+```
+```sh
+# expected output
+service/api-server-svc created
+endpoints/api-server-svc created
+```
+---
+### Access the VM workload via the `Ingress Loadbalancer`
+
+Retrieve the public IP address of the `Ingress LB` and try accessing it via
+a browser.
+
+```sh
+INGRESS_IP=$(kubectl get ingress/pos-ingress -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+echo $INGRESS_IP
+```
+```sh
+34.117.58.199 # you might have a different IP address
+```
+<p align="center">
+  <img src="docs/images/pos.png">
+  <em>
+    </br>
+    (click image to enlarge)
+  </em>
+</p>
+
+---
+### Clean up
+
+When you delete the VM, the associated resources are also deleted except for the
+`DataVolume` that gets created to import/host the VM image. You can manually
+delete this resource. You can also use it to mount as a boot volume into new VM.
+
+```sh
+kubectl virt delete vm pos-vm
+
+# if you want to also delete the datavolume
+kubectl delete dv pos-vm-dv
```

Comment:
@Shabirmean I think this should be `kubectl delete dv pos-vm-boot-dv`?

Reply:
Fixed in https://github.com/GoogleCloudPlatform/anthos-samples/pull/255/commits/1b21d191f2b3ddeeb42f2853b6d895307af9010e